shell.nix

```diff
@@ -1,17 +1,17 @@
 {
   /* This should match Nixpkgs commit in status-mobile. */
   source ? builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.tar.gz";
-    sha256 = "sha256:0mwck8jyr74wh1b7g6nac1mxy6a0rkppz8n12andsffybsipz5jw";
+    url = "https://github.com/NixOS/nixpkgs/archive/ddf0003c57fb5cbb4a9754f2f6d5ebe9cdae5151.tar.gz";
```

Comment:
Is this commit going to be used in `status-mobile` as well?

Reply:
Should be! i copied it from https://github.com/status-im/status-mobile/pull/18321 so it probably makes sense to merge both this PR and that one once approved

Reply:
hmm 
we cant use this url -> https://github.com/NixOS/nixpkgs/archive/ddf0003c57fb5cbb4a9754f2f6d5ebe9cdae5151.tar.gz since its from nixpkgs master branch, what I had in the status-mobile PR back then was an experiment :D 

Here is what I'll do next week : 
- open PR with this status-go version in status-mobile to see what breaks and what doesn't.
- we'll verify if any `nixpkgs` stable version 23.11 got the fixes we want related to `GraalVM` or not ( else we just fork and move on )

and then what url we decide on at status-mobile we can use that here.

Reply:
actually for now you could just use : `e7603eba51f2c7820c0a182c6bbb351181caa8e7`
since we do have go 1.20 over here -> https://github.com/NixOS/nixpkgs/blob/e7603eba51f2c7820c0a182c6bbb351181caa8e7/pkgs/development/compilers/go/1.20.nix